Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -929,7 +932,11 @@ open class NavigationMapView: UIView {
                     
                     try mapView.mapboxMap.style.addSource(arrowStrokeSource, id: NavigationMapView.SourceIdentifier.arrowStrokeSource)
                     arrowStrokeLayer.source = NavigationMapView.SourceIdentifier.arrowStrokeSource
-                    try mapView.mapboxMap.style.addLayer(arrowStrokeLayer, layerPosition: .above(mainRouteLayerIdentifier))
+                    if allLayerIds.contains(mainRouteLayerIdentifier) {
```

Comment:
The CarPlay's issue with maneuvers should be related with the layerID problem here. Sometimes the `mapView` could not find the `mainRouteLayerIdentifier`. Thus add the check of layer existence before setting layer position. Similar issue happened for the waypoint circle layer.